electrum/gui/qt/utxo_list.py

```diff
@@ -96,6 +96,8 @@ def insert_utxo(self, idx, utxo: PartialTxInput):
         amount = self.parent.format_amount(utxo.value_sats(), whitespaces=True)
         labels = [name_short, address, label, amount, '%d'%height]
         utxo_item = [QStandardItem(x) for x in labels]
+        # Storing the full outpoint as the data payload of this item
+        utxo_item[0].setData(name)
```

Comment:
There are already a few similar lines below, in particular `utxo_item[self.Columns.ADDRESS].setData(name, Qt.UserRole)`.
Let's move this below that one.
Also, like those lines below, please do not use a magic number here (`0`).
Along with the suggestion for the other file, we could have:
```suggestion
        utxo_item[self.Columns.OUTPOINT].setData(name, self.ROLE_CLIPBOARD_DATA)
```

Reply:
Also done.